packages/playground/src/App.tsx

```diff
@@ -1,12 +1,15 @@
-import Wasm from './Wasm';
-import Welcome from './Welcome';
+import { Layout } from './components/Layout';
```

Comment:
I think the components folder should become an index.ts file, but this can be addressed in another PR